client/client.go

```diff
@@ -234,7 +234,6 @@ func loadCredentials(opts *withCredentials) (grpc.DialOption, error) {
 			return nil, errors.Wrap(err, "could not read certificate/key")
 		}
 		cfg.Certificates = []tls.Certificate{cert}
-		cfg.BuildNameToCertificate()
```

Comment:
This has been deprecated since go 1.14 and no longer necessary. https://pkg.go.dev/crypto/tls@go1.14#Config.BuildNameToCertificate

Reply:
It started showing up as an error during lint once I updated golangci-lint

Reply:
Fix this with another PR

Reply:
Created #3023